verifier/src/main/java/com/aws/jverify/verifier/compiler/JavaFrontEnd.java

```diff
@@ -136,9 +136,11 @@ public void finished(TaskEvent e) {
                     // Apply the second half of our pipeline as above (4 and onwards).
                     // See the implementation of JavaCompiler.compile() for similar lines,
                     // including the comment "these method calls must be chained to avoid memory leaks"
-                    envs.addAll(unsuspend(lower(suspend(
-                            unsubstitute(unlambda(substitute(
-                                    compiler.flow(compiler.attribute(todo)))))))));
+                    envs.addAll(unsuspend(
+                            lower(suspend(
+                                    unlambda(
+                                    compiler.flow(compiler.attribute(todo))
```

Comment:
```suggestion
                    envs.addAll(
                            unsuspend(lower(suspend(
                                    unlambda(
                                            compiler.flow(compiler.attribute(todo))
```

Or potentially other ways, but I think at least keeping `unsuspend(lower(suspend(` together makes sense.